src/pages/nowplaying/style.css

```
@@ -0,0 +1,81 @@
+h1 {
+  font-size: 24px;
+  margin-top: 24px;
+}
+
+#now-playing #hero-image {
+  width: 100%;
+  height: 316px;
+  margin-top: 20px;
+}
+#now-playing #title-like {
```

Seria bom adicionar um enter para manter a boa indentação 
```suggestion

#now-playing #title-like {
```